src/pem_fmt_plug.c

```diff
@@ -0,0 +1,423 @@
+/* PEM (PKCS #8) cracker.
+ *
+ * This software is Copyright (c) 2015, Dhiru Kholia <kholia at kth.se>,
+ * and it is hereby released to the general public under the following terms:
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted.
+ *
+ * This code may be freely used and modified for any purpose.
+ *
+ * Big thanks to Martin Kleppmann, and Lapo Luchini for making this format
+ * possible.
+ */
+
+#if FMT_EXTERNS_H
+extern struct fmt_main fmt_pem;
+#elif FMT_REGISTERS_H
+john_register_one(&fmt_pem);
+#else
+
+#include <string.h>
+#include <assert.h>
+#include <errno.h>
+#include <openssl/des.h>
+#ifdef _OPENMP
+#include <omp.h>
+#ifndef OMP_SCALE
+#define OMP_SCALE               64
+#endif
+#endif
+
+#include "arch.h"
+#include "misc.h"
+#include "common.h"
+#include "formats.h"
+#include "params.h"
+#include "options.h"
+#include "johnswap.h"
+#include "pbkdf2_hmac_sha1.h"
+#include "jumbo.h"
+#include "memdbg.h"
+#include "asn1.h"
+
+#define FORMAT_LABEL            "PEM"
+#define FORMAT_NAME             "PEM"
```

Comment:
Whenever FORMAT_NAME is same as FORMAT_LABEL, the former should be replaced with empty string.